webpack.config.dev.js

```diff
@@ -10,8 +11,12 @@ export default {
     // To support react-hot-loader
     alias: {
       '~': path.resolve(__dirname, './src'),
+      '~images': path.resolve(__dirname, './images'),
       'react-dom': '@hot-loader/react-dom',
     },
+    plugins: [
```

Comment:
Any specific reason for using this plugin?

Reply:
It's for `import Path from "~/Some/Dir/Path/ndex.js"` -> `import Path from "~/Some/Dir/Path"`.  Or is there any workaround to make this work without using this plugin?

Reply:
I mean I didnt see such usages 👀

Reply:
It's for this part 
https://github.com/gogumaa/zeppelin-web/pull/12/files#diff-f880b57906a072cdffa62da2c954ddd4R12

Reply:
ah, cool, the weird thing is that I have never noticed this thing in my projects :)

Reply:
Right I know what you mean. I actually was about to ask you lol 
I also couldn't find where we make this possible in Zepl webapp  webpack configuration 😅